types/rtree/tsconfig.json

```diff
@@ -14,7 +14,8 @@
         ],
         "types": [],
         "noEmit": true,
-        "forceConsistentCasingInFileNames": true
+        "forceConsistentCasingInFileNames": true,
+        "esModuleInterop": true
```

Comment:
why do you need `esModuleInterop` here?

Reply:
Hi @gabritto, without `esModuleInterop` I get...
```
Module '"/home/chris/propeller/git/temp/DefinitelyTyped/types/rtree/index"' can only be default-imported using the 'esModuleInterop' flag
```

If I switch from `export = rtree` to `export default rtree` I get...

```
ERROR: 1:1   npm-naming  The declaration doesn't match the JavaScript module 'rtree'. Reason:
The declaration should use 'export =' syntax because the JavaScript source uses 'module.exports =' syntax and 'module.exports' can be called or constructed.

To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
If you won't fix this error now or you think this error is wrong,
you can disable this check by adding the following options to your project's tslint.json file under "rules":

    "npm-naming": [true,{"mode":"code","errors":[["NeedsExportEquals",false]]}]
 See: https://github.com/Microsoft/dtslint/blob/master/docs/npm-naming.md
ERROR: 24:1  npm-naming  The declaration doesn't match the JavaScript module 'rtree'. Reason:
The declaration specifies 'export default' but the JavaScript source does not mention 'default' anywhere.

The most common way to resolve this error is to use 'export =' syntax instead of 'export default'.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
If you won't fix this error now or you think this error is wrong,
you can disable this check by adding the following options to your project's tslint.json file under "rules":

    "npm-naming": [true,{"mode":"code","errors":[["NoDefaultExport",false]]}]
 See: https://github.com/Microsoft/dtslint/blob/master/docs/npm-naming.md
```

Reply:
It looks like the import should be `import RTree = require('rtree');`